src/networks.js

```diff
@@ -80,11 +116,12 @@ export function getNetwork(chainId = getPreferredChain()) {
 }
 
 export function getEthersNetwork() {
-  const { type, chainId, ensRegistry } = getNetwork()
+  const { type, chainId, ensRegistry, defaultEthNode } = getNetwork()
   return {
     name: type,
     chainId: chainId,
     ensAddress: ensRegistry,
+    _defaultProvider: defaultEthNode,
```

Comment:
Are we going to need this in the end? Thought that we saw that it expects a function